javascript-objects/main.js

```diff
@@ -0,0 +1,31 @@
+const student = {
+  firstName: 'Mohamed',
+  lastName: 'Elmessiry',
+  age: 34
+};
+const fullname = (student.firstName) + ' ' + (student.lastName);
+console.log('value of fullname: ' + ' ' + fullname);
+(student.livesInIrvine) = false;
+(student.previousOccupation) = 'Financial Analyst';
+console.log('value of student lives in Irvine:' + ' ' + student.livesInIrvine);
+console.log('value of previous occupation:' + ' ' + student.previousOccupation);
+console.log(student);
+
+const vehichle = {
+  make: 'mercedes',
+  model: 'gullwing',
+  year: '1957'
+};
+(vehichle.color) = 'silver';
+(vehichle.isConvertible) = false;
```

Comment:
Please remove the parentheses, they are unnecessary

Reply:
![Screen Shot 2021-01-26 at 11 14 22 AM](https://user-images.githubusercontent.com/69738204/105871848-b6817480-5fc7-11eb-95a3-e74b1b5f33d5.png)